mne/viz/evoked.py

```diff
@@ -1198,3 +1199,437 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
+        be a dict of legal inputs to `matplotlib.pyplot.plot`. These
+        parameters will be passed to the line plot call of the corresponding
+        condition, overriding defaults.
+        E.g., if evokeds is a dict with the keys "Aud/L", "Aud/R",
+        "Vis/L", "Vis/R", `styles` can be `{"Aud/L":{"linewidth":1}}` to set
+        the linewidth for "Aud/L" to 1. Note that HED ('/'-separated) tags are
+        not supported.
+    vlines : list of int
+        A list of integers corresponding to the positions, in seconds,
+        at which to plot dashed vertical lines.
+    ci : float | None
+        If not None and `evokeds` is a [list/dict] of lists, a confidence
+        interval is drawn around the individual time series. This value
+        determines the CI width. E.g., if this value is .95 (the default),
+        the 95% parametric confidence interval is drawn.
+        If None, no shaded confidence band is plotted.
+    truncate_yaxis : bool
+        If True, the left y axis is truncated to half the max value and
+        rounded to .25 to reduce visual clutter. Defaults to True.
+    ylim : dict | None
+        ylim for plots (after scaling has been applied). e.g.
+        ylim = dict(eeg=[-20, 20])
+        Valid keys are eeg, mag, grad, misc. If None, the ylim parameter
+        for each channel equals the pyplot default.
+    invert_y : bool
+        If True, negative values are plotted up (as is sometimes done
+        for ERPs out of tradition). Defaults to False.
+    ax : None | `matplotlib.pyplot.axes` instance | list of `axes`
+        What axes to plot to. If None, a new axes is created.
+        When plotting multiple channel types, can also be a list of axes, one
+        per channel type.
+    title : None | str
+        If str, will be plotted as figure title. If None, the channel
+        names will be shown.
+    show : bool
+        If True, show the figure.
+
+    Returns
+    -------
+    fig : Figure
+        The figure in which the plot is drawn.
+    """
+    import matplotlib.pyplot as plt
+    from ..evoked import Evoked
+
+    # set up labels and instances
+    if isinstance(evokeds, Evoked):
+        conditions = ['Evoked']  # empty title
+        evokeds = dict(Evoked=evokeds)
+    else:
+        if isinstance(evokeds, dict):
+            conditions = sorted(list(evokeds.keys()))
+        else:
+            evokeds = dict((str(ii + 1), evoked)
+                           for ii, evoked in enumerate(evokeds))
+            conditions = sorted(list(evokeds.keys()))
+
+    # get and set a few limits and variables (times, channels, units)
+    example = (evokeds[conditions[0]]
+               if isinstance(evokeds[conditions[0]], Evoked)
+               else evokeds[conditions[0]][0])
+    if not isinstance(example, Evoked):
+            raise ValueError("evokeds must be an `mne.Evoked` "
+                             "or of a collection of `mne.Evoked`s")
```

Comment:
They go in places that Sphinx will render, like docstrings. Not error messages.

This is also over-indented